pkg/apis/core/v1alpha1/logicalcluster_types.go

```diff
@@ -0,0 +1,175 @@
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	conditionsv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/apis/conditions/v1alpha1"
+	"github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/util/conditions"
+)
+
+// LogicalCluster describes the current workspace.
```

Comment:
We mix usage of LogicalCluster and Workspace frequently here and elsewhere. Do we need to be more diligent about keeping them separate?

Reply:
yes, we have to

Reply:
done here